themes/aquablank/layouts/partials/page_cloudsploit.html

```diff
@@ -63,6 +70,57 @@ <h1 class="title page_title is-spaced fadeInUp animationDelay_1">{{ .Params.disp
             </div><!-- column is-8 -->
 
             <div class="column is-4-desktop is-6-tablet is-12-mobile fadeInUp animationDelay_2">
+                <div class="round-aera">
+                    <div class="box is-rounded has-text-danger	 is-size-3">
+                        High
+                    </div>
```

Comment:
We need to pull this from the page not hardcode as different things will have different severity classification.